images/10.x-master/alpine/Dockerfile

```diff
@@ -183,6 +183,7 @@ ENV DOCKER_DB_TIMEOUT=120 \
     FRAPPE_DEFAULT_SITE= \
     FRAPPE_HTTP_PORT=80 \
     FRAPPE_WEBSERVER_PORT=80 \
+    FRAPPE_SOCKETIO_PORT=3000 \
```

Comment:
Just for my understanding.
Why do we keep the default value 3000 for the dockerfile but we use the prod value for the docker-compose ?

Reply:
Well, my idea was to keep the Dockerfile as close as possible to Frappe default values, but use the docker-compose as an example on how to configure a "_standard_" production environment, but now you're making me doubt...